Microsoft.Toolkit.Win32.WpfCore.SampleApp/MainWindow.xaml.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Windows;
+using Microsoft.Toolkit.Uwp.Notifications;
+
+namespace Microsoft.Toolkit.Win32.WpfCore.SampleApp
+{
+    /// <summary>
+    /// Interaction logic for MainWindow.xaml
+    /// </summary>
+    public partial class MainWindow : Window
+    {
+        public MainWindow()
+        {
+            InitializeComponent();
+        }
+
+        private void ButtonShowToast_Click(object sender, RoutedEventArgs e)
+        {
+            new ToastContentBuilder()
```

Comment:
Could be handy to have a few more examples here of showing how to do things like customize the title or add action buttons and showing the application receiving those events too.

Reply:
Beefed up the example to show a fully interactive toast with inline reply, viewing an image, etc!